domain/service/service_inf/invitation_service_inf.go

```diff
@@ -6,9 +6,13 @@ import (
 )
 
 type IInvitationService interface {
-	CreateGroupInvitation(request GroupInputDTO, groupID UUID) error
+	CreateGroupInvitation(request GroupInvitationDTO) error
 
 	AcceptGroupInvitation(invitation UUID, userID UUID) error
 
 	DeclineGroupInvitation(invitation UUID, userID UUID) error
+
+	GetGroupInvitationByID(id UUID) (GroupInvitationOutputDTO, error)
+
+	GetGroupInvitationsFromUser(id UUID) ([]GroupInvitationOutputDTO, error)
```

Comment:
Just a quick convenience thing:

1. Can we switch the parameter name in "GetGroupInvitationByID" to invitationId and in "GetGroupInvitationsFromUser" to userID?

2. I like "GetGroupInvitationsByUser" better (that's what it's called in storage too). It feels like it puts the focus on the user, showing we're grabbing stuff linked to them. What do you think?